plugin.video.hak5/addon.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+import os
+import sys
+import urlparse
+import xbmc
+import xbmcaddon
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon(id="plugin.video.hak5").getAddonInfo('path'), 'resources', 'lib'))
```

Comment:
I'm not sure how that works. Instead of doing this:
LIB_DIR = xbmc.translatePath(
    os.path.join(xbmcaddon.Addon(id="plugin.video.hak5").getAddonInfo('path'), 'resources', 'lib'))
sys.path.append(LIB_DIR)

from hak5_const import ADDON, DATE, VERSION

Can you tell me what i should do instead?